new-website/src/components/location/edit-page.jsx

```diff
@@ -9,7 +9,9 @@ import { LOCATION_COLLECTION } from "."
 
 export const EditPage = ({ docId, onError }) => {
     const { user, partnerConfigs, isLoading } = useSession()
-    console.log(user, partnerConfigs, isLoading) // <- all undefined
+    const { partnerIds } = user
+
+    console.log(partnerIds, partnerConfigs, isLoading) // <- now getting `null, null, true`
```

Comment:
Log output looks like this:
```
AuthContextProvider Dispatched Action
  Object
    payload: {}
    state: {user: null, isAuthenticated: false, isLoading: true}
    type: "initializing"
    __proto__: Object

AuthContextProvider Dispatched Action
  Object
    payload: {user: P, isAuthenticated: true}
    state: {user: null, isAuthenticated: false, isLoading: true}
    type: "login"
    __proto__: Object

null null true

UserContext Dispatched Action loadingUser
  Object
    payload: {}
    state: {isLoadingUser: true, isLoadingPartner: true, partnerIds: null, partnerId: null, partnerConfigs: null, …}
    type: "loadingUser"
    __proto__: Object

null null true
```

Reply:
everything works as expected ... it is still loading the stuff ... see this screenshot 
<img width="1100" alt="image" src="https://user-images.githubusercontent.com/6809130/86638083-f4409080-bfd6-11ea-8e82-ff7600ab6808.png">

Reply:
Yes, `edit` works but `add` doesn't

Reply:
`add` now works. Faced a combination of issues, but the essential one here was that firestore rules prevented me from creating new database entries without passing a real `partnerId` (I did just use "test") for a while.. It took some time to figure out. See the change log for more fixes in the area.